apps/src/block_utils.js

```diff
@@ -545,14 +546,30 @@ const LABELED_INPUT_PARTS_REGEX = /(.*?)({([^}]*)}|\n|$)/m;
  * Finds the input config for the given input name, and removes it from args.
  * @param {InputConfig[]} args List of configs to search through
  * @param {string} inputName name of input to find and remove
+ * @param {string} blockText original block text used for metrics reporting in the case of a missing input
  * @returns InputConfig the input config with name `inputName`
  */
-const findAndRemoveInputConfig = (args, inputName) => {
-  const argIndex = args.findIndex(arg => arg.name === inputName);
-  if (argIndex !== -1) {
-    return args.splice(argIndex, 1)[0];
+const findAndRemoveInputConfig = (args, inputName, blockText) => {
+  if (args.length === 0) {
```

Comment:
Nice check! What do you think of adding a console error here to alert developer locally? I guess they would variable label on the block locally.

Reply:
Yep, `MetricsReporter` will automatically log to the console in local development (rather than actually posting to the metrics endpoint)